public/app/features/dashboard/state/DashboardModel.ts

```diff
@@ -1156,6 +1157,21 @@ export class DashboardModel implements TimeModel {
     }
   }
 
+  toggleExemplarsForAll() {
+    for (const panel of this.panels) {
+      for (const target of panel.targets) {
+        if (!(target.datasource && target.datasource.type === 'prometheus')) {
+          continue;
+        }
+
+        const promTarget = target as PromQuery;
```

Comment:
This is the assertion that is failing the betterer check in drone, you can run yarn betterer -u and commit the results to get the tests passing.

Reply:
Thanks, drone is happy now.